src/shared/organisms/DataPanel/styles.less

```diff
@@ -4,7 +4,7 @@
   position: fixed;
   bottom: 0;
   width: 100%;
-  z-index: 4;
+  z-index: 9999;
```

Comment:
is this style still needed?

Reply:
yeah, feedback from Simon, that the data panel should be on top of the `ResourceView` (Resource Gallery Page)